### PR TITLE
CosmosDB Emulator support

### PR DESCRIPTION
**Pull request by javaadpatel**
Commit Messages:
-----------
added UseCosmosDbEmulator extension method
updated readme to link to setting up cosmosDB emulator

### CODE REVIEW

File: ExRam.Gremlinq.Providers.CosmosDb/GremlinQuerySourceExtensions.cs [hunk 1]
```diff
@@ -134,5 +134,26 @@ public static IConfigurableGremlinQuerySource UseCosmosDb(this IConfigurableGrem
                         source.Logger)
                     .UseGraphsonDeserialization(new TimespanConverter()));
         }
+
+        public static IConfigurableGremlinQuerySource UseCosmosDbEmulator(this IConfigurableGremlinQuerySource source, string database, string graphName, string authKey, string hostname = "localhost", int port = 8901)
```
Comment (danielcweber) [inline ExRam.Gremlinq.Providers.CosmosDb/GremlinQuerySourceExtensions.cs]: That would just work fine, thanks a lot. It is also a great chance for some code reuse ;-)

Comment (javaadpatel) [reply ExRam.Gremlinq.Providers.CosmosDb/GremlinQuerySourceExtensions.cs]: I updated this method for code reuse, is this better?

File: ExRam.Gremlinq.Providers.CosmosDb/GremlinQuerySourceExtensions.cs [hunk 2]
```diff
@@ -155,5 +134,16 @@ public static IConfigurableGremlinQuerySource UseCosmosDbEmulator(this IConfigur
...
                    .UseGraphsonDeserialization(new TimespanConverter()));
         }
+
+        public static IConfigurableGremlinQuerySource UseCosmosDb(this IConfigurableGremlinQuerySource source, string hostname, string database, string graphName, string authKey, int port = 443)
+        {
+            return source.CreateCosmosDBGremlinQuerySource(hostname, database, graphName, authKey, port, true);
+        }
+
+
+        public static IConfigurableGremlinQuerySource UseCosmosDbEmulator(this IConfigurableGremlinQuerySource source, string database, string graphName, string authKey, string hostname = "localhost", int port = 8901)
```
Comment (danielcweber) [inline ExRam.Gremlinq.Providers.CosmosDb/GremlinQuerySourceExtensions.cs]: Almost there. I'd like to have the host name without default value please.

**Comment by danielcweber** [comment]
Thanks a lot!

**Pull request merged**